pkg/datapath/linux/config/config.go

```diff
@@ -223,6 +223,9 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 
 	if option.Config.EnableSRv6 {
 		cDefinesMap["ENABLE_SRV6"] = "1"
+		if option.Config.SRv6Mode == "reduced" {
```

Comment:
Subjective but, I think this would be cleaner if we aligned the defaults with the datapath flags. 

Our default is "reduced", so IMO it would be cleaner to swap this clause around to:

```
		if option.Config.SRv6Mode == "srh" {
			cDefinesMap["ENABLE_SRV6_SRH_ENCAP"] = "1"
		}
```

Then the datapath can read "if SRH encap enabled, do this...", and not "if reduced is not enabled".